.github/workflows/release.yaml

```diff
@@ -15,20 +15,20 @@ jobs:
     - uses: actions/checkout@v2
     - run: git fetch --prune --unshallow
     
-    #Creates and Pushes the git tag, Download's the hyscale binary from the PROD and pointing it to the latest, Copying the hyscale.jar from the image for upoading the artifact while publishing the release, Note: pom_version and RELEASE_VERSION should always be the same(Else when tag is created it will be created from the master)
+    # Download's the hyscale binaries from Dev Latest and pointing it to the Prod latest
     - name: release_info
       id: release_info
       run: |
          . .github/workflows/release_info.sh
          echo ::set-output name=CHANGELOG::$(echo -e "${CHANGELOG}")
-         wget https://s3-$AWS_REGION.amazonaws.com/$AWS_DEV_S3_BUCKET/hyscale/release/latest/hyscale -O /var/tmp/hyscale
-         IMAGE_VERSION=`cat /var/tmp/hyscale | grep HYSCALE_BUILD_VERSION= | awk -F "=" '{print $2}'`
-         docker run -v $(pwd):/var/tmp --entrypoint /bin/cp  hyscale/hyscale:$IMAGE_VERSION /usr/local/bin/hyscale.jar /var/tmp
-         . ./scripts/publish_artifacts.sh
-         echo ::set-output name=RELEASE_VERSION::$(echo $project_version)
+         for binary in hyscale win/hyscale.exe mac/hyscale hyscale.jar
+         do
+           curl https://s3-$AWS_REGION.amazonaws.com/$AWS_DEV_S3_BUCKET/hyscale/release/latest/$binary --create-dirs -o /var/tmp/$binary
+           aws s3 cp /var/tmp/$binary s3://$AWS_S3_BUCKET/hyscale/release/latest/$binary
+           aws s3api put-object-tagging --bucket $AWS_S3_BUCKET  --key hyscale/release/latest/$binary --tagging 'TagSet=[{Key=hyscalepubliccontent,Value=true}]'
+         done
+         echo ::set-output name=RELEASE_VERSION::$(cat pom.xml | grep "<version>.*</version>" | head -1 |awk -F'[><]' '{print $3}')
          echo ::set-output name=RELEASE_NAME::$(cat pom.xml | grep "<release.name>.*</release.name>" | head -1 |awk -F'[><]' '{print $3}')
```

Comment:
why do we need  the same variable setting two times ?

Reply:
One is RELEASE_NAME and other is RELEASE_VERSION